android/modules/ui/src/java/ti/modules/titanium/ui/widget/listview/ListViewProxy.java

```diff
@@ -379,11 +402,21 @@ private void processProperty(String name, Object value)
 
 			// Set list sections.
 			setSections((Object[]) value);
-		}
 
-		if (name.equals(TiC.PROPERTY_EDITING)) {
+		} else if (name.equals(TiC.PROPERTY_EDITING)) {
+			final TiViewProxy parent = getParent();
+
+			if (parent != null) {
+
+				// Due to Android limitations, selection trackers cannot be removed.
+				// Re-create ListView with new selection tracker.
+				releaseViews();
+				parent.add(this);
+			}
+
+		} else if (name.equals(TiC.PROPERTY_SHOW_SELECTION_CHECK)) {
 
-			// Update list.
+			// Update and refresh list.
 			update();
 		}
 	}
```

Comment:
Calling `selectItem()` will throw a `NullPointerException` if property "editing" is set `false`.
_(Does not crash on iOS.)_